mmv1/templates/terraform/encoders/compute_target_https_proxy.go.erb

```diff
@@ -7,4 +7,12 @@ if _, ok := obj["certificateManagerCertificates"]; ok {
 	obj["sslCertificates"] = obj["certificateManagerCertificates"]
 	delete(obj, "certificateManagerCertificates")
 }
-return obj, nil
\ No newline at end of file
+
+// Send null if serverTlsPolicy is not set. Without this, Terraform would not send any value for `serverTlsPolicy`
+// in the "PATCH" payload so if you were to remove a server TLS policy from a target HTTPS proxy, it would NOT remove
```

Comment:
What is the behavior today. If we remove it when it's not set I would expect a persistent diff.

Reply:
So the behavior today is that if you remove `server_tls_policy`, it is completely excluded from the payload in the `PATCH` request, so Google simply leaves the `serverTlsPolicy` set as it was. This change makes it so that if `server_tls_policy` is not set, then `serverTlsPolicy` will be `null` in the `PATCH` payload, which actually removes the association to the server TLS policy (the behavior we want).